cwms-data-api/src/main/java/cwms/cda/api/MeasurementController.java

```diff
@@ -0,0 +1,247 @@
+    public void getOne(@NotNull Context ctx, @NotNull String locationId) {
+        try (final Timer.Context ignored = markAndTime(GET_ONE)) {
+            throw new UnsupportedOperationException(NOT_SUPPORTED_YET);
+        }
+
+    }
+
+    @OpenApi(
+            requestBody = @OpenApiRequestBody(
+                    content = {
+                            @OpenApiContent(isArray = true, from = Measurement.class, type = Formats.JSONV1),
+                            @OpenApiContent(from = Measurement.class, type = Formats.JSON)
+                    },
+                    required = true),
+            queryParams = {
+                    @OpenApiParam(name = FAIL_IF_EXISTS, type = Boolean.class,
+                            description = "Create will fail if provided Measurement(s) already exist. Default: true")
+            },
+            description = "Create new measurement(s).",
+            method = HttpMethod.POST,
+            tags = {TAG},
+            responses = {
+                    @OpenApiResponse(status = "204", description = "Measurement(s) successfully stored.")
+            }
+    )
+    @Override
+    public void create(Context ctx) {
+
+        try (Timer.Context ignored = markAndTime(CREATE)) {
+            String formatHeader = ctx.req.getContentType();
+            ContentType contentType = Formats.parseHeader(formatHeader, Measurement.class);
+            List<Measurement> measurements = parseMeasurements(ctx, contentType);
+            boolean failIfExists = ctx.queryParamAsClass(FAIL_IF_EXISTS, Boolean.class).getOrDefault(true);
+            DSLContext dsl = getDslContext(ctx);
+            MeasurementDao dao = new MeasurementDao(dsl);
+            if(measurements.size() == 1) {
+                dao.storeMeasurement(measurements.get(0), failIfExists);
+                ctx.status(HttpServletResponse.SC_CREATED).json("Created Measurement");
+            } else {
+                dao.storeMeasurements(measurements, failIfExists);
+                ctx.status(HttpServletResponse.SC_CREATED).json("Created Measurements");
+            }
+        }
+    }
+
+    @OpenApi(ignore = true)
+    @Override
+    public void update(@NotNull Context ctx, @NotNull String locationId) {
+        try (final Timer.Context ignored = markAndTime(GET_ONE)) {
+            throw new UnsupportedOperationException(NOT_SUPPORTED_YET);
+        }
+    }
+
+    @OpenApi(
+            queryParams = {
+                    @OpenApiParam(name = OFFICE, required = true, description = "Specifies the office of the measurements to delete"),
+                    @OpenApiParam(name = MIN_NUMBER, description = "Specifies the min number-id of the measurement to delete."),
+                    @OpenApiParam(name = MAX_NUMBER, description = "Specifies the max number-id of the measurement to delete."),
+                    @OpenApiParam(name = MIN_DATE, description = "Specifies the minimum date (in ISO-8601 format) of the measurement to delete."),
```

Comment:
maintain consistency with dates in the other API's by providing a time zone

Reply:
added timezone param

Reply:
the schema descriptions should be consistent for dates and time zone. it has precise language.

Reply:
Made descriptions consistent with other uses of date and tz params. I also updated `min_date` and `max_date` to be `begin` and `end` for consistency.